models/marts/aggregates/agg_yearly_parts_hardcoded_pivot.sql

```diff
@@ -23,7 +23,7 @@ select
     * 
 from
     merged
-    -- have to manually map strings in the pivot operation
+    -- Ref: https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax#pivot_operator
```

Comment:
@sdurry Not sure why the shift to the link?

Reply:
Oh damn 🤦, was meant to be an addition. I will change it back.